<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupDialog</class>
 <widget class="QDialog" name="SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	font-family: &quot;consolas&quot;;
	color: black;
}

QRadioButton::indicator
{
	color: #edd0be;
}

QGroupBox
{
	border: 1px solid #edd0be;
}

QLineEdit
{
	border:	1px solid #edd0be;
	height: 25px;
}

QPushButton
{
	border-radius: 15px;
	background-color: #edd0be;
}

QPushButton:hover
{
	background-color: #ffcea6;
}

#SetupDialog
{
	background-color: white;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>layout</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>red:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="rlayoutText">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(224, 54, 54, .3);</string>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>blue:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="blayoutText">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(37, 198, 252, .3);</string>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
